### PR TITLE
feat: SQL chain security disclaimers to documentation & JSDoc

### PR DESCRIPTION
**Pull request by bracesproul**

### CODE REVIEW

File: docs/docs/modules/chains/popular/sqlite.mdx [hunk 1]
```diff
@@ -45,3 +45,20 @@ const db = await SqlDatabase.fromDataSourceParams({
 If desired, you can return the used SQL command when calling the chain.
 
 <CodeBlock language="typescript">{SqlDBSqlOutputExample}</CodeBlock>
+
+# Disclaimer ⚠️
+
+The query chain may generate insert/update/delete queries. When this is not expected, use a custom prompt or create a SQL users without write permissions.
```
Comment (jacoblee93) [inline docs/docs/modules/chains/popular/sqlite.mdx:51]: Grammatical mistake

**Review by jacoblee93 — COMMENTED** [review]
Small typo

**Comment by langchain-infra** [comment]
https://langchainjs-docs-pbhko38m3-langchain.vercel.app

File: docs/docs/modules/chains/popular/sqlite.mdx (hunk 1 above)
Comment (bracesproul) [reply docs/docs/modules/chains/popular/sqlite.mdx]: Resolved

**Comment by jacoblee93** [comment]
Thank you!

**Pull request merged**